app/helpers/application_helper.rb

```diff
@@ -12,7 +12,7 @@ def published_document_path(document)
   end
 
   def state(document)
-    state = document.publication_state
+    state = document.publication_state == "live" ? "published" : document.publication_state
```

Comment:
this could alternatively be `state = document.publication_state.gsub("live", "published")` which is a bit pithier but may not be as clear...